Patch for the fact we are using the mingw-w64 with mingw32 compiler

diff -ur dcmtk-3.6.0.orig/ofstd/include/dcmtk/ofstd/offile.h dcmtk-3.6.0/ofstd/include/dcmtk/ofstd/offile.h
--- dcmtk-3.6.0.orig/ofstd/include/dcmtk/ofstd/offile.h	2014-06-30 12:23:34.314366353 -0400
+++ dcmtk-3.6.0/ofstd/include/dcmtk/ofstd/offile.h	2014-06-30 12:29:57.079959273 -0400
@@ -81,7 +81,7 @@
   typedef __int64 offile_off_t;
   typedef fpos_t offile_fpos_t;
 #else
-  #ifdef EXPLICIT_LFS_64
+  #if defined(EXPLICIT_LFS_64) && !defined(__MINGW64__) && !defined(__MINGW32__)
     // Explicit LFS (LFS64)
     typedef fpos64_t offile_fpos_t;
     typedef off64_t offile_off_t;
@@ -196,7 +196,7 @@
   OFBool popen(const char *command, const char *modes)
   {
     if (file_) fclose();
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(__MINGW64__) && !defined(__MINGW32__)
     file_ = _popen(command, modes);
 #else
     file_ = :: popen(command, modes);
@@ -258,7 +258,7 @@
     {
       if (popened_)
       {
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(__MINGW64__) && !defined(__MINGW32__)
         result = _pclose(file_);
 #else
         result = :: pclose(file_);
